docs/DEVELOPER_GUIDE.md

```diff
@@ -193,12 +186,35 @@ this:
     └── kubectl-foo-windows.exe
 ```
 
-The resulting installation directory would up just with:
-
-```text
-.
-└── krew-foo-windows.exe
-```
+- To copy all files in the `bin/` directory of the extracted archive to
+  the root of your plugin, use the default:
+
+  ```yaml
+      files: []  # same as [{from: "*", to: "."}]
```

Comment:
Hmm. I think empty list should _not_ be treated as unspecified (`nil`) field.

Empty means do nothing, which a valid operation, but it shouldn't be allowed since it does not make sense.

Reply:
Isn't the differentiation between empty and nil a language subtlety? From a user's perspective both cases mean "no rules given". And here I think we are complicating things if we make that distinction.

Reply:
I'll move this to main thread.